tests/unit/core/test_deploy_export.py

```diff
@@ -317,28 +320,37 @@ def test_hf_bert(self):
                 ("attention_mask", torch.randint(low=0, high=2, size=(2, 16)).cuda()),
             ]
         )
-        self.__test_export_route_all(module=bert, out_name="bert", input_example=input_example)
+        self.__test_export_route(module=bert, out_name="bert", mode=df_type, input_example=input_example)
 
     @pytest.mark.unit
     @pytest.mark.run_only_on('GPU')
-    def test_jasper_encoder(self):
+    @pytest.mark.parametrize(
+        "df_type", [DF.ONNX, DF.TORCHSCRIPT, DF.PYTORCH, pytest.param(DF.TRTONNX, marks=requires_trt)]
+    )
+    def test_jasper_encoder(self, df_type):
         with open("tests/data/jasper_smaller.yaml") as file:
             yaml = YAML(typ="safe")
             jasper_model_definition = yaml.load(file)
 
         jasper_encoder = nemo_asr.JasperEncoder(
             conv_mask=False,
             feat_in=jasper_model_definition['AudioToMelSpectrogramPreprocessor']['features'],
-            **jasper_model_definition['JasperEncoder']
+            **jasper_model_definition['JasperEncoder'],
         )
 
```

Comment:
Ok, so now I see that this could be rewritten using the import_from_config. Not crucial though...